### ISSUE TITLE
Grails compatibility

### ISSUE THREAD

**Issue by sophiaraw** [groovydev/twitter-bootstrap-grails-plugin#134]
Does this plugin work with grails 3.0.3? If not, which is the most recent version that is compatible with the plugin?

**Comment by bdbogjoe** [comment]
Actually there no official version for grails 3

There us one pull request for this : https://github.com/groovydev/twitter-bootstrap-grails-plugin/pull/131

**Comment by hansgru** [comment]
> There us one pull request for this : #131 

Yes, but please keep the compatibility with Grails 2.5.x too. There are way too many applications that can't upgrade yet.
